explainer.md

```diff
@@ -0,0 +1,386 @@
+
+## Proposed Solution: Secure Payment Confirmation
+
+Secure Payment Confirmation (SPC) builds on top of [WebAuthn] to add
+payment-specific datastructures to the signed assertion, and to relax
+assumptions to allow the API to be called in payment contexts.
+
+The intention for Secure Payment Confirmation is that a Customer would enroll
+once on a given device for a given account with an Account Provider, either on
+the Account Provider's website or during a traditionally-authenticated online
+payment (e.g. after completing a challenge in a Account Provider iframe). Then,
+in subsequent transactions on **any Merchant** that wishes to use Secure
+Payment Confirmation:
+
+1. The Customer identifies themselves to the Merchant as per normal (e.g.
+   by selecting a payment instrument such as a credit card or bank account.)
+1. Using a back-channel (e.g. the EMV® 3-D Secure protocol), the Merchant asks
+   for, and receives, a list of credentials for the identified Customer from
+   the Account Provider.
+1. The Merchant calls the SPC API with the list of
+   [webauthn-credentials|credentials].
+1. The User Agent displays a UX to the Customer, informing them of the
+   transaction details and asking if they wish to authenticate their identity to
+   the Account Provider.
+1. The Customer consents, and the User Agent and Customer perform a [WebAuthn]
+   signing ceremony. Payment details are included in the returned assertion.
+1. The Merchant receives the assertion, and using the existing back-channel
+   sends the assertion to the Account Provider.
+1. The Account Provider verifies the signature on the assertion, and verifies
+   that the data in the assertion (e.g. transaction amount, payee) is as
+   expected.
+1. The Account Provider informs the Merchant of transaction success, and the
+   payment concludes successfully.
+
+> **NOTE**: Most of the above flow happens in the background. The user
+> experience consists only of examining and agreeing to the transaction
+> details, and performing a [WebAuthn] interaction.
+
+**TODO**: Color-coded image here showing the authentication flow from the user
+perspective. Merchant site with 'pay' button, then overlaid payment transaction
+details, then WebAuthn interface, then success.
+
+For the handling of cases when no returned credentials match the current
+device, see [the Privacy section](#privacy-considerations).
+
+### Proposed APIs
+
+Secure Payment Confirmation introduces a new [WebAuthn extension], `payment`,
+which adds three payments-specific capabilities on top of traditional WebAuthn:
+
+1. Allows calling `navigator.credentials.create` in a cross-origin iframe, as long
+   as a ["payment" permission policy] is set on the iframe.
+1. Allows a third-party (the Merchant) to initiate an authentication ceremony
+   **on behalf of** the RP (the Account Provider), by passing in credentials
+   provided to the Merchant by the Account Provider.
```

Comment:
I suggest: "(typically provided to the Merchant by the Account Provider)." I don't want to rule out delegation use cases.